src/MaciVoting.sol

```diff
@@ -35,14 +37,19 @@ contract MaciVoting is PluginUUPSUpgradeable, ProposalUpgradeable, IMaciVoting {
     using SafeCastUpgradeable for uint256;
 
     /// @notice The [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID of the contract.
-    bytes4 internal constant MACI_VOTING_INTERFACE_ID =
-        this.initialize.selector ^ this.getVotingToken.selector;
+    bytes4 internal constant MACI_VOTING_INTERFACE_ID = this.initialize.selector
+        ^ this.minProposerVotingPower.selector ^ this.totalVotingPower.selector
+        ^ this.getVotingToken.selector ^ this.minParticipation.selector ^ this.minDuration.selector
+        ^ this.getProposal.selector ^ this.changeCoordinatorPublicKey.selector;
 
     /// @notice The ID of the permission required to call the
     /// `changeCoordinatorPublicKey` function.
     bytes32 public constant CHANGE_COORDINATOR_PUBLIC_KEY_PERMISSION_ID =
         keccak256("CHANGE_COORDINATOR_PUBLIC_KEY_PERMISSION_ID");
 
+    /// @notice The factor to scale down the voice credits.
+    uint256 public constant VOICE_CREDIT_FACTOR = 10e16;
```

Comment:
should this be a constant? what if the governance token is 10e6 and everyone ends up having 0 voting power due to scaling down?

Reply:
ah ok good point, I just saw it a magic var so put it as as constant. What do you think this should be - A constructor arg? A function arg?

Reply:
could it be https://github.com/privacy-scaling-explorations/maci-voting-plugin-aragon/blob/main/src/MaciVoting.sol#L176 .decimals() stored in the initialize function?

Reply:
If I am understanding correctly, this `factor` is used to assign voting power based on the number of tokens a user has?

So it could theoretically be 1 token = 1 voice credit, or it could be 1 token = 10 voice credits - depending on the `factor`

If we want the number of tokens to exactly match the number of voice credits, then sounds like we can use the decimals value safely. If we want to allow more configurations than a 1-1 mapping, then having the `factor` configurable would be needed right?

Leaning towards keeping it simple and mandating 1 token = 1 voice credit - and in that case using decimals - wdyt?

Reply:
Ok sweet, I will do that

Reply:
Done - I will add a test in a follow up PR if that is ok?